validator/sawtooth_validator/journal/chain.py

```diff
@@ -13,6 +13,7 @@
 # limitations under the License.
 # ------------------------------------------------------------------------------
 
+# pylint: disable=too-many-lines
```

Comment:
This should really be disabled in pylintrc, not per-file.  This must have been re-enabled accidentally or due to changes in pylint.